Topics/Development_Process/Github_Pages.md

```diff
@@ -0,0 +1,91 @@
+# GitHub Pages
+
+If you have ever created a "site" before using HTML, maybe with CSS and perhaps Javascript, and wondered how you could host it online through a link, here's your solution.
+
+From the [GitHub docs](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages):
+
+> GitHub Pages is a static site hosting service that takes HTML, CSS, and JavaScript files straight from a repository on GitHub, optionally runs the files through a build process, and publishes a website.
+
+In other words, it is a **free static website host.** It's reliable, your pages will remain up forever (unless GitHub changes this), and can handle a reasonable amount of bandwidth (100GB/mo), given you aren't serving images or large files.
+
+**Note:** Free for public repositories. You need GitHub pro if you want to do this for private repositories. The deployed "page" will be public but your repo will remain private. Students should be able to get Pro for free.
+
+## Caveats
+
+Many people use GitHub pages to host blogs and personal portfolios. You can also host web applications, although I would only suggest using it for lightweight stuff -- mainly, things that would be **read**, not **written.** In other words, I **would not** use GitHub pages as a substitute for Google Forms, even though you could still do that. **I would also not use GitHub pages if you need to pair a frontend with a backend.** There are other great alternatives that you can find in this wiki, such as [Vercel](https://vercel.com/), [Render](https://render.com/), and so on.
+
+**Beware:** [GitHub doesn't like using people using pages for](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#prohibited-uses) commercial use or to handle sensitive data.
+
+## Getting started - Deploying a static HTML site
+
+Here's a quick overview on **how to deploy a simple, static HTML site.** It's simple, but it can be hard to remember clearly, so feel free to use this as a reference every time you do so.
+
+1. Create a new repo.
+2. Add your HTML/CSS/js files to your repository.
+   1. You should name your HTML file `index.html` -- that's the "home" page.
+   2. Other than that, if your css/javascript works on your machine if you were to clone this repo, it should also work on GitHub pages.
```

Comment:
We should keep capitalization consistent for "Github pages": maybe we can capitalize the word "pages" throughout this article as "Github Pages", since it refers to the official name for a service.